google/resumable_media/requests/download.py

```diff
@@ -19,6 +19,9 @@
 from google.resumable_media.requests import _helpers
 
 
+_SINGLE_GET_CHUNK_SIZE = 8192
```

Comment:
Does this translate into an HTTP request boundary or does it just represent how much data we're pulling off the wire at once?